package.json

```diff
@@ -1,29 +1,12 @@
 {
-  "dependencies": {
```

Comment:
keeping dependencies here is nice because we can do

`import 'bootbox' from 'bootbox'` in functions like `BootboxWrapper`